src/app/machines/components/MachineHeaderForms/ActionFormWrapper/ActionFormWrapper.test.tsx

```diff
@@ -16,42 +14,31 @@ import {
   rootState as rootStateFactory,
   tagState as tagStateFactory,
 } from "testing/factories";
+import { renderWithBrowserRouter } from "testing/utils";
 
 const mockStore = configureStore();
 
-it("can set selected machines to those that can perform action", async () => {
-  const state = rootStateFactory();
-  const machines = [
-    machineFactory({ system_id: "abc123", actions: [NodeActions.ABORT] }),
-    machineFactory({ system_id: "def456", actions: [] }),
-  ];
-  const store = mockStore(state);
-  render(
-    <Provider store={store}>
-      <MemoryRouter
-        initialEntries={[{ pathname: "/machines", key: "testKey" }]}
-      >
-        <CompatRouter>
-          <ActionFormWrapper
-            action={NodeActions.ABORT}
-            clearHeaderContent={jest.fn()}
-            machines={machines}
-            viewingDetails={false}
-          />
-        </CompatRouter>
-      </MemoryRouter>
-    </Provider>
-  );
+afterEach(() => {
+  jest.restoreAllMocks();
```

Comment:
`restoreAllMocks`  restores all mocks back to their original value (and `global.scrollTo` still will be overwritten with a mock function in other tests).

Might be a better idea to restore the original implementation instead to clean this up properly.

If your intention was to mock `scrollTo` in all tests in this file, it should probably be made clear by setting it up in `beforeAll`.

Reply:
This should probably be done for each test as `scrollTo` is not implemented in js-dom so this could error for each test. I'll fix that up.